src/Entity/CreditMemoChannel.php

```diff
@@ -28,4 +32,9 @@ public function name(): string
     {
         return $this->name;
     }
+
+    public function color(): string
```

Comment:
Spec needs to be adjusted as well https://github.com/Sylius/RefundPlugin/blob/master/spec/Entity/CreditMemoChannelSpec.php